<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArticleAmountControl</class>
 <widget class="QWidget" name="ArticleAmountControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_tabIgnore">
      <attribute name="title">
       <string>Ignoring old articles</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="MultiFeedEditCheckBox" name="m_mcbAddAnyDateArticles"/>
         </item>
         <item>
          <widget class="QCheckBox" name="m_cbAddAnyDateArticles">
           <property name="text">
            <string>Add articles with any date into the database</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="MultiFeedEditCheckBox" name="m_mcbAvoidOldArticles"/>
         </item>
         <item>
          <widget class="QWidget" name="m_wdgAvoidOldArticles" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="m_gbAvoidOldArticles">
              <property name="title">
               <string>Avoid adding articles before this date/time into the database</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="0">
                <widget class="QRadioButton" name="m_rbAvoidAbsolute">
                 <property name="text">
                  <string>Absolute date/time</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QRadioButton" name="m_rbAvoidRelative">
                 <property name="text">
                  <string>Relative time</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="1">
                <widget class="QDateTimeEdit" name="m_dtDateTimeToAvoid">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumDate">
                  <date>
                   <year>1971</year>
                   <month>1</month>
                   <day>1</day>
                  </date>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="1">
                <widget class="TimeSpinBox" name="m_spinHoursAvoid">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <double>10000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="HelpSpoiler" name="m_helpIgnoring" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabLimit">
      <attribute name="title">
       <string>Limiting amount of article in feeds</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="MultiFeedEditCheckBox" name="m_mcbArticleLimittingCustomize"/>
         </item>
         <item>
          <widget class="QCheckBox" name="m_cbArticleLimittingCustomize">
           <property name="text">
            <string>Customize article limits</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="MultiFeedEditCheckBox" name="m_mcbArticleLimittingSetup"/>
         </item>
         <item>
          <widget class="QWidget" name="m_wdgArticleLimittingCustomize" native="true">
           <layout class="QFormLayout" name="formLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>In database, keep</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="m_spinArticleCount">
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QCheckBox" name="m_cbNoRemoveImportant">
              <property name="text">
               <string>Do not remove important articles</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="m_cbNoRemoveUnread">
              <property name="text">
               <string>Do not remove unread articles</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QCheckBox" name="m_cbMoveToBinNoPurge">
              <property name="text">
               <string>Just move articles to recycle bin, do not purge them</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="HelpSpoiler" name="m_helpLimit" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>timespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>MultiFeedEditCheckBox</class>
   <extends>QCheckBox</extends>
   <header>multifeededitcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>HelpSpoiler</class>
   <extends>QWidget</extends>
   <header>helpspoiler.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_rbAvoidAbsolute</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_dtDateTimeToAvoid</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_rbAvoidRelative</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_spinHoursAvoid</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
